src/libmongoc/tests/test-mongoc-structured-log.c

```diff
@@ -714,4 +823,12 @@ test_structured_log_install (TestSuite *suite)
    TestSuite_Add (suite, "/structured_log/duration", test_structured_log_duration);
    TestSuite_Add (suite, "/structured_log/level_names", test_structured_log_level_names);
    TestSuite_Add (suite, "/structured_log/component_names", test_structured_log_component_names);
+   TestSuite_Add (suite, "/structured_log/max_document_length", test_structured_log_max_document_length);
+   TestSuite_AddFull (suite,
+                      "/structured_log/env_defaults",
+                      test_structured_log_env_defaults,
+                      NULL,
+                      NULL,
+                      test_structured_log_skip_if_env_not_default,
+                      NULL);
```

Comment:
```suggestion
                      test_structured_log_skip_if_env_not_default);
```
The `TestSuite_AddFull` macro (not to be confused with the `_TestSuite_AddFull` function it wraps) guarantees a `NULL` terminating argument is passed at the end of the argument list.

Reply:
My mistake.